src/AddressUkPafLookupViewBridge.js

```diff
@@ -0,0 +1,139 @@
+var bridge = function (presenterPath) {
+    window.rhubarb.viewBridgeClasses.HtmlViewBridge.apply(this, arguments);
+};
+
+bridge.prototype = new window.rhubarb.viewBridgeClasses.HtmlViewBridge();
+bridge.prototype.constructor = bridge;
+
+bridge.prototype.attachEvents = function() {
+
+    var self = this,
+        alertClass = "c-alert",
+        manualAddressElements = $(".manual-fields"),
+        searchAddressElement = $(".search-fields"),
+        insertManualAddressLink = $(".manual-address-link"),
+        manualAddressPar = $(".manual-address-par"),
+        searchLink = $(".search-address-link"),
+        houseNumber = self.findChildViewBridge('HouseNumber'),
+        postCodeSearch = self.findChildViewBridge('PostCodeSearch'),
+        searchError = $(".search-error"),
+        searchResultsMsg = $(".search-results-msg"),
+        resultItemsList = $(".search-results-items"),
+        searchButton = self.findChildViewBridge('Search'),
+        // address fields
+        line1 = self.findChildViewBridge('Line1'),
+        line2 = self.findChildViewBridge('Line2'),
+        town = self.findChildViewBridge('Town'),
+        county = self.findChildViewBridge('County'),
+        postCode = self.findChildViewBridge('PostCode');
+
+    // if the's a post code we suppose that there's an address set
+    if(postCode.viewNode.value != '') {
+        showAddressFields();
+    } else {
+        // default configuration
+        manualAddressElements.hide();
+        searchLink.hide();
+        searchError.hide();
+    }
+
+    // address manual entry
+    insertManualAddressLink.click(function() {
+        showAddressFields()
+    });
+    // search address
+    searchLink.click(function() {
+        resultItemsList.empty();
+        manualAddressPar.show();
+        manualAddressElements.hide();
+        searchAddressElement.show();
+        searchLink.hide();
+    });
+
+    // search address
+    searchButton.attachClientEventHandler("OnButtonPressed", function() {
+        searchResultsMsg.removeClass(alertClass).empty();
```

Comment:
This hides the result count message, but leaves the previous list of results visible.